src/coreComponents/common/format/table/TableData.hpp

```diff
@@ -112,9 +128,26 @@ class TableData
     return getCellsData() == comparingTable.getCellsData();
   }
 
+  /**
+   * @brief Get all error messages
+   * @return The vector of error messages
```

Comment:
Could be something else than a vector (a `std::set`, a `std::map`, an LVArray...)
```suggestion
   * @return The list of error messages
```

Reply:
changed